src/Components/Chat/Chatbox.tsx

```diff
@@ -10,44 +10,45 @@ import {
   StyledTextField,
   TextBoxContainer
 } from './StyledComponents'
-import { getRandom, wordbank } from '../../../public/wordbank'
 
 import type { MessageT } from "./Types";
 import SendIcon from '@mui/icons-material/Send';
 import socket from "../Websocket/socket";
 
 type OwnPropsT = {
-  roomName: string | null
+  roomName: string | null,
+  currentWord?: string
 }
 
-const Chatbox = ({ roomName }: OwnPropsT): ReactElement => {
+const Chatbox = ({ roomName, currentWord='' }: OwnPropsT): ReactElement => {
   const sessionId = sessionStorage.getItem('sessionId')
   const [messages, setMessages] = useState<MessageT[]>([])
   const [currentMessage, setCurrentMessage] = useState("")
-  // const [gamewords, setGameWords] = useState<string[]>([])
-  const [currWord, setCurrWord] = useState("")
   
   useEffect(()=>{
-    const words = getRandom(5, wordbank)
-    setCurrWord(words[Math.floor(Math.random()*words.length)])
-    
     socket.on("message_to_client", ({author, message})=>{
       setMessages((prevMessages) =>{
         return [...prevMessages, {author, message}]
       })
     })
+
+    socket.on('player_guessed_correct_word', (username) => {
+      setMessages([...messages, {author: "Server", message: `${username} guessed the correct word!`}])
+    })
     
```

Comment:
Right now, the current drawer can guess the word also, we should do a check to make sure the the person who types a message isn't the current drawer.